src/Ship.cpp

```diff
@@ -1215,11 +1215,10 @@ void Ship::StaticUpdate(const float timeStep)
 				// after the whole physics update, which means the flight state on next
 				// step would be HYPERSPACE, thus breaking quite a few things.
 				LuaEvent::Queue("onLeaveSystem", this);
-			}
-			//give player some time to check the wheels
-			else if (m_hyperspace.countdown <= 0.0f && is_equal_exact(m_wheelState,1.0f)) {
+			} 
+      		else if (m_wheelState != 0.0f) {
```

Comment:
Not to torture you,  but don't you get compiler warning for this, assuming you're on GNU/Linux using GCC? See #4090.